### PR TITLE
serializers account app

### PR DESCRIPTION
**Pull request by SharProd**
add serializers, endpoints,urls in account app

### CODE REVIEW

File: Eduplatform_site/account/endpoints.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from .models import User,Teacher,Student,Group
+
+
+class UserListAPIView(ListCreateAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+class UserDetailApi(RetrieveUpdateDestroyAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
```
Comment (nikitapt63) [inline Eduplatform_site/account/endpoints.py]: why ApiView not ViewSet?

File: Eduplatform_site/account/endpoints.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+class StudentDetailApi(RetrieveUpdateDestroyAPIView):
+    queryset = Student.objects.all()
+    serializer_class = StudentSerializer
+
+
+class GroupListAPIView(ListCreateAPIView):
+    queryset = Group.objects.all()
+    serializer_class = GroupSerializer
+
+
```
Comment (nikitapt63) [inline Eduplatform_site/account/endpoints.py:36]: Add 2 endpoints for all students/teachers from one group

File: Eduplatform_site/account/serializers.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
+from rest_framework import serializers
+from .models import User,Teacher,Student,Group
+
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id','first_name','last_name','email']
```
Comment (nikitapt63) [inline Eduplatform_site/account/serializers.py]: make it in  tuple

File: Eduplatform_site/account/urls.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+from .endpoints import (
+    UserListAPIView,
+    UserDetailApi,
+    TeacherListAPIView,
+    TeacherDetailApi,
+    StudentListAPIView,
+    StudentDetailApi,
+    GroupListAPIView,
+    GroupDetailApi)
+
```
Comment (nikitapt63) [inline Eduplatform_site/account/urls.py:11]: may be better to use routers and view set?

**Review by nikitapt63 — CHANGES_REQUESTED** [review]
Answer questions and add endpoints.

File: Eduplatform_site/account/endpoints.py (hunk 2 above)
Comment (SharProd) [reply Eduplatform_site/account/endpoints.py:36]: students and teachers are visible when viewing the group

**Pull request merged**